src/coreclr/tools/Common/Compiler/HardwareIntrinsicHelpers.cs

```diff
@@ -73,17 +73,13 @@ private static class XArchIntrinsicConstants
             public const int Avx512f = 0x8000;
             public const int Avx512f_vl = 0x10000;
```

Comment:
Can we just have this be `Avx512vl`, as that's the actual CPUID feature name and matches the casing we use elsewhere?

Reply:
I agree, will rename this part accordingly.